demos/bridge/primefaces3-portlet/src/main/webapp/views/applicant.xhtml

```diff
@@ -66,7 +66,7 @@
 									<f:selectItem itemLabel="#{i18n['select']}" itemValue="" />
 									<f:selectItems itemValue="#{province.provinceId}" itemLabel="#{province.provinceName}"
 										value="#{listModelBean.provinces}" var="province" />
-									<p:ajax update="provinceIdField" />
+									<f:ajax render="provinceIdField" />
 								</h:selectOneMenu>
```

Comment:
Okay this pr has been updated. I needed to add `p:ajax` to keep the fancy Ajax validation working for the state selection.